client/components/Queries.tsx

```diff
@@ -22,7 +22,49 @@ export interface IProps {
 
 // eslint-disable-next-line react/function-component-definition
 const Queries: React.FC<IProps> = ({ rawQueries }) => {
+    // "rawQueries" is the raw, unfilter array of queries. "listOfQueries" is the filter list
+    const [listOfQueries, setListOfQueries] = useState(rawQueries);
+
+    // State for the list
+    const [list, setList] = useState([...listOfQueries.slice(0, 150)]);
+
+    // State to trigger oad more
+    const [loadMore, setLoadMore] = useState(false);
+
+    // State of whether there is more to load
+    const [hasMore, setHasMore] = useState(listOfQueries.length > 150);
+
+    // Load more button click
+    const handleLoadMore = () => {
+        setLoadMore(true);
+    };
     /** State requirments for this component */
+    useEffect(() => {
+        /** once the projects have loadend and a project has been selected, send the query to get queres for the project */
+        setListOfQueries(rawQueries);
+    }, [rawQueries]);
+
+    useEffect(() => {
+        setList(listOfQueries.slice(0, 150));
```

Comment:
Let's create a variable for 150 to avoid magic numbers being used in multiple places.

Reply:
done